cli/src/main/java/org/infinispan/cli/impl/StreamShell.java

```diff
@@ -78,6 +79,11 @@ public Key read() throws InterruptedException {
       }
    }
 
+   @Override
+   public Key read(long timeout, TimeUnit unit) throws InterruptedException {
+      return read();
```

Comment:
Do we need to worry here not abiding by the timeout? Or I guess every InputStream provided here is already completed?

Reply:
I simply mirrored what the default implementation of this does. TBH, I'm not even sure this method is being invoked.

Reply:
Sure, integrating.